osf_tests/test_preprint_summary.py

```diff
@@ -19,11 +18,33 @@ def preprint_provider():
 def preprint(preprint_provider):
     return PreprintFactory._build(PreprintService, provider=preprint_provider)
 
+
+@pytest.fixture()
+def yesterday_at_midnight():
+    return datetime.datetime.utcnow().replace(hour=0,
```

Comment:
Fun sidenote, `utcnow` does not actually give it tzinfo, but `timezone.now()` does.
<img width="472" alt="screen shot 2017-08-24 at 4 52 25 pm" src="https://user-images.githubusercontent.com/1322421/29688160-ba1ff16c-88ec-11e7-8a73-dbc1379915c3.png">